stunnel/stunnel.ml

```diff
@@ -26,6 +26,7 @@ exception Stunnel_error of string
 exception Stunnel_verify_error of string
 
 let certificate_path = "/etc/stunnel/certs"
```

Comment:
If this path is not used at all by `Stunnel` it should be removed from here and added to the places that use it (e.g. XAPI) to avoid confusion that changing this directory has any effect on stunnel.

Reply:
Yes, I will change the `certificate_path` into xapi instead.